Add entries for Azure CLI

CarnegieJ opened the pull request:
Added comparable Azure CLI commands that correspond with other respective ways to manage Azure resources.

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

CarnegieJ committed:
Add entries for Azure CLI

Added comparable Azure CLI commands that correspond with other respective ways to manage Azure resources.
